src/quest/manager.py

```diff
@@ -151,3 +166,25 @@ async def get_event(self, workflow_id: str, name: str, identity: str | None) ->
     async def get_identity_queue(self, workflow_id: str, name: str, identity: str | None) -> IdentityQueue:
         await self._check_resource(workflow_id, name, identity)
         return self._wrap(IdentityQueue(), workflow_id, name, identity)
+
+    async def register_alias(self, alias: str, workflow_id: str):
```

Comment:
Let's make these private: `_register_alias` and `_deregister_alias`

We don't want developers calling these directly.

Reply:
How can I protected these and also call them in the manager_wrappers?